requirements.txt

```diff
@@ -1,9 +1,13 @@
 git+https://github.com/rwbarton/django.git@59f1fc4c7b65d23fceffd5c87e2b6ea350030c56
+Flask==0.10.1
```

Comment:
can you add a comment next to these new deps like this:
 `Flask==0.10.1 # Dependency of moto s3 mock`
to make it clear these are only needed in testing?  I think in the not too distant future we're going to be splitting out the things only needed in development from requirements.txt and it'd be great to have these tagged.